groovy/postInit/electronics/ElectronicCircuits.groovy

```diff
@@ -10,38 +12,60 @@ mods.gregtech.assembler.removeByInput(30, [metaitem('wireFineCopper') * 4, metai
+// Resistor * 2
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCharcoal'), metaitem('wireFineCopper') * 4], [fluid('glue') * 100])
+// Resistor * 2
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCarbon'), metaitem('wireFineCopper') * 4], [fluid('glue') * 100])
+// Resistor * 4
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCoal'), metaitem('wireFineAnnealedCopper') * 4], [fluid('glue') * 100])
+// Resistor * 4
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCharcoal'), metaitem('wireFineAnnealedCopper') * 4], [fluid('glue') * 100])
+// Resistor * 4
+mods.gregtech.assembler.removeByInput(6, [metaitem('dustCarbon'), metaitem('wireFineAnnealedCopper') * 4], [fluid('glue') * 100])
+
+crafting.removeByOutput(metaitem('component.resistor')) 
+
+carbons = new ItemStack[]{
+        metaitem('dustCoal'),
+        metaitem('dustCharcoal'),
+        metaitem('dustCarbon'),
+        metaitem('dustCoal'),
+        metaitem('dustHighPurityCarbon'),
+        metaitem('dustAnthracite'),
+        metaitem('dustCoal')
+}
 
-//Remove steel plates from electronic circuits since they were unnecessary
-
-crafting.replaceShaped("gregtech:electronic_circuit_lv", metaitem('circuit.electronic'), [
-        [metaitem('component.resistor'), ore('craftingToolWireCutter'), metaitem('component.resistor')],
-        [metaitem('circuit.vacuum_tube'), metaitem('circuit_board.basic'), metaitem('circuit.vacuum_tube')],
-        [ore('cableGtSingleRedAlloy'), ore('cableGtSingleRedAlloy'), ore('cableGtSingleRedAlloy')]])
-
-crafting.replaceShaped("gregtech:electronic_circuit_mv", metaitem('circuit.good_electronic'), [
-        [metaitem('component.diode'), ore('craftingToolWireCutter'), metaitem('component.diode')],
-        [metaitem('circuit.electronic'), metaitem('circuit_board.good'), metaitem('circuit.electronic')],
-        [ore('wireGtSingleCopper'), metaitem('circuit.electronic'), ore('wireGtSingleCopper')]])
-
-crafting.addShaped('resistor_wire_anthracite', metaitem('component.resistor') * 2, [
-    [metaitem('rubber_drop'),item('minecraft:paper'),metaitem('rubber_drop')],
-    [ore('wireGtSingleCopper'),ore('dustAnthracite'), ore('wireGtSingleCopper')],
-    [null,item('minecraft:paper'),null]])
-
-crafting.addShaped('resistor_wire_fine_anthracite', metaitem('component.resistor') * 2, [
-    [metaitem('rubber_drop'),item('minecraft:paper'),metaitem('rubber_drop')],
-    [ore('wireFineCopper'),ore('dustAnthracite'), ore('wireFineCopper')],
-    [null,item('minecraft:paper'),null]])
-	
-crafting.addShaped('resistor_wire_coke', metaitem('component.resistor') * 2, [
-    [metaitem('rubber_drop'),item('minecraft:paper'),metaitem('rubber_drop')],
-    [ore('wireGtSingleCopper'),ore('dustCoke'), ore('wireGtSingleCopper')],
-    [null,item('minecraft:paper'),null]])
+mods.gregtech.assembler.recipeBuilder()
+        .fluidInputs(fluid('glue') * 100)
+        .input(new GTRecipeItemInput(carbons, 1))
+        .inputs(ore('wireFineCopper') * 4)
```

Comment:
You probably also want to include normal-thickness wire recipes, at least for copper.

Reply:
base CEu doesn't have assembler recipes using normal-thickness wire?

Reply:
Oh, I think I meant to comment on the crafting recipes, since there isn't one of those anymore.

Reply:
Uhm there is I think, it uses an or (either single gt or fine wire) since in vanilla gt neither gives a bonus over the other